src/link/index.ts

```diff
@@ -360,12 +385,30 @@ async function linkAllModules (
   )
 }
 
-async function linkPkg (
+async function reflinkPkg (
   filesResponse: PackageFilesResponse,
   dependency: DependencyTreeNode,
   opts: {
     force: boolean,
     baseNodeModules: string,
+    packageImportMethod: 'auto' | 'hardlink' | 'copy' | 'reflink',
+  }
+) {
+  const pkgJsonPath = path.join(dependency.hardlinkedLocation, 'package.json')
+
+  if (!filesResponse.fromStore || opts.force || !await exists(pkgJsonPath)) {
+    await execFilePromise('mkdir', ['-p', dependency.hardlinkedLocation])
```

Comment:
You can use the mkdirp lib here. It is more reliable because it retries on fails.